fs.go

```diff
@@ -105,6 +105,11 @@ var (
 	rootFSHandler RequestHandler
 )
 
+// PathNotFound fires when file is not found in filesystem
+// this functions tries to replace "Cannot open requested path"
+// server response giving to the programmer the control of server flow.
+type PathNotFoundFunc func(ctx *RequestCtx)
```

Comment:
I don't think this is needed. This is exactly the same type as [RequestHandler](https://github.com/erikdubbelboer/fasthttp/blob/82bb5fe32e0a93ab983224863e8d826d7b0ae6a5/server.go#L127).

Reply:
This function is to handle FS structure when do not find requested file.